src/main/scala/Validator.scala

```diff
@@ -91,9 +110,22 @@ class Validator(var tableCsvFile: URI, sourceUri: String = "") {
     errors = errors.concat(e)
   }
 
+  private def fetchPrimaryKeyString(list: List[Any]): String = {
+    var primaryKeyString = ""
+    for (i <- list) {
+      i match {
+        case listOfAny: List[Any] =>
+          for (j <- listOfAny) primaryKeyString += (j.toString + ",")
```

Comment:
Scala has a nice function `mkString` to do this kind of thing for you.

Reply:
ok